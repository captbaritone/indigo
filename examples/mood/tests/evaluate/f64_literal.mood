pub fn test(): f64 {
  10.00_f64
}