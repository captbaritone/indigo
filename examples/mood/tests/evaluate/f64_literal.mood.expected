-----------------
INPUT
----------------- 
pub fn test(): f64 {
  10.00_f64
}
-----------------
OUTPUT
-----------------
10